ldap-connector/processes/writer.mod

```diff
@@ -0,0 +1,116 @@
+[Ivy]
+17D75C0DFF198A83 9.3.1 #module
+>Proto >Proto Collection #zClass
+wr0 writer Big #zClass
+wr0 B #cInfo
+wr0 #process
+wr0 @AnnotationInP-0n ai ai #zField
+wr0 @TextInP .type .type #zField
+wr0 @TextInP .processKind .processKind #zField
+wr0 @TextInP .xml .xml #zField
+wr0 @TextInP .responsibility .responsibility #zField
+wr0 @StartSub f0 '' #zField
+wr0 @EndSub f1 '' #zField
+wr0 @GridStep f3 '' #zField
+wr0 @PushWFArc f2 '' #zField
+wr0 @StartSub f5 '' #zField
+wr0 @CallSub f9 '' #zField
+wr0 @PushWFArc f11 '' #zField
+wr0 @PushWFArc f4 '' #zField
+wr0 @PushWFArc f6 '' #zField
+>Proto wr0 wr0 writer #zField
+wr0 f0 inParamDecl '<String distinguishedName,javax.naming.directory.Attributes attributes> param;' #txt
+wr0 f0 inParamInfo 'attributes.description=attributes that define the new object
+distinguishedName.description=unique name of the new object' #txt
+wr0 f0 inParamTable 'out.action="create";
+out.attributes=param.attributes;
+out.distinguishedName=param.distinguishedName;
+' #txt
+wr0 f0 outParamDecl '<> result;' #txt
+wr0 f0 callSignature create(String,javax.naming.directory.Attributes) #txt
+wr0 f0 @CG|tags connector #txt
+wr0 f0 @C|.xml '<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<elementInfo>
+    <language>
+        <name>create(String,Attributes)</name>
+    </language>
+</elementInfo>
+' #txt
+wr0 f0 81 49 30 30 -13 17 #rect
+wr0 f0 res:/webContent/icons/ldap.png?small #fDecoratorIcon
+wr0 f1 649 65 30 30 0 15 #rect
+wr0 f3 actionTable 'out=in;
+' #txt
+wr0 f3 actionCode 'import com.axonivy.connector.ldap.LdapWriter;
+
+LdapWriter writer = new LdapWriter(in.jndiConfig);
+
+if(in.action.equalsIgnoreCase("create")){
+	writer.createObject(in.distinguishedName, in.attributes);
+}
+else if(in.action.equalsIgnoreCase("destroy")){
```

Comment:
my personal preference would be to use the word "delete". because this is more common (e.g. CRUD create read update delete).

but that's your decision!

Reply:
thanks, agreed. is it possible to read the signature of the start event? then no variable is needed...

Reply:
as far as I know this is not possible